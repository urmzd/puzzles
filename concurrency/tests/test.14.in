TRX 11235 Al_Alpha Beth_Beta 150000 40
TRX 11236 Cat_Charlie Dov_Delta 150000 20
TRX 11237 Eve_Echo Fin_Foxtrot 150000 25
TRX 11238 Guy_Golf Hana_Hotel 150000 40
TRX 11239 Iona_Indigo Jeff_Juliet 150000 20
TRX 11240 Kit_Kilo Liam_Lima 150000 25
TRX 11241 Mona_Mike Nick_November 150000 40
TRX 11242 Otto_Oscar Peta_Papa 150000 20
TRX 11243 Qu_Quebec Reya_Romeo 150000 25
EPOCH
BLK 1 0 0x00000000 6
11235 Al_Alpha Beth_Beta 150000 40
11238 Guy_Golf Hana_Hotel 150000 40
11241 Mona_Mike Nick_November 150000 40
11236 Cat_Charlie Dov_Delta 150000 20
11237 Eve_Echo Fin_Foxtrot 150000 25
11239 Iona_Indigo Jeff_Juliet 150000 20
0x00000044 0x0031353d
MINE 8
END
