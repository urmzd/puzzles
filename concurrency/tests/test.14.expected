Received event TRX with ID 11235
Adding transaction: 11235 Al_Alpha Beth_Beta 150000 40
Received event TRX with ID 11236
Adding transaction: 11236 Cat_Charlie Dov_Delta 150000 20
Received event TRX with ID 11237
Adding transaction: 11237 Eve_Echo Fin_Foxtrot 150000 25
Received event TRX with ID 11238
Adding transaction: 11238 Guy_Golf Hana_Hotel 150000 40
Received event TRX with ID 11239
Adding transaction: 11239 Iona_Indigo Jeff_Juliet 150000 20
Received event TRX with ID 11240
Adding transaction: 11240 Kit_Kilo Liam_Lima 150000 25
Received event TRX with ID 11241
Adding transaction: 11241 Mona_Mike Nick_November 150000 40
Received event TRX with ID 11242
Adding transaction: 11242 Otto_Oscar Peta_Papa 150000 20
Received event TRX with ID 11243
Adding transaction: 11243 Qu_Quebec Reya_Romeo 150000 25
Received event EPOCH
Aging transaction (2):11235 Al_Alpha Beth_Beta 150000 40
Aging transaction (1):11236 Cat_Charlie Dov_Delta 150000 20
Received event BLK with ID 1
Removing transaction: 11235 Al_Alpha Beth_Beta 150000 40
Removing transaction: 11238 Guy_Golf Hana_Hotel 150000 40
Removing transaction: 11241 Mona_Mike Nick_November 150000 40
Removing transaction: 11236 Cat_Charlie Dov_Delta 150000 20
Removing transaction: 11237 Eve_Echo Fin_Foxtrot 150000 25
Removing transaction: 11239 Iona_Indigo Jeff_Juliet 150000 20
Received event MINE
Block mined: 2 1 0x0031353d 3
11240 Kit_Kilo Liam_Lima 150000 25
11242 Otto_Oscar Peta_Papa 150000 20
11243 Qu_Quebec Reya_Romeo 150000 25
Thread 0 checking nonce 0x00000000
Thread 0 checking nonce 0x00000008
Thread 0 checking nonce 0x00000010
Thread 0 checking nonce 0x00000018
Thread 0 checking nonce 0x00000020
Thread 0 checking nonce 0x00000028
Thread 0 checking nonce 0x00000030
Thread 0 checking nonce 0x00000038
Thread 0 checking nonce 0x00000040
Thread 0 checking nonce 0x00000048
Thread 0 checking nonce 0x00000050
Thread 0 checking nonce 0x00000058
Thread 0 checking nonce 0x00000060
Thread 0 checking nonce 0x00000068
Thread 2 checking nonce 0x00000002
Thread 0 checking nonce 0x00000070
Thread 1 checking nonce 0x00000001
Thread 1 checking nonce 0x00000009
Thread 5 checking nonce 0x00000005
Thread 2 checking nonce 0x0000000a
Thread 5 checking nonce 0x0000000d
Thread 2 checking nonce 0x00000012
Thread 5 checking nonce 0x00000015
Thread 5 checking nonce 0x0000001d
Thread 1 checking nonce 0x00000011
Thread 1 checking nonce 0x00000019
Thread 4 checking nonce 0x00000004
Thread 7 checking nonce 0x00000007
Thread 4 checking nonce 0x0000000c
Thread 7 checking nonce 0x0000000f
Thread 5 checking nonce 0x00000025
Thread 7 checking nonce 0x00000017
Thread 5 checking nonce 0x0000002d
Thread 3 checking nonce 0x00000003
Thread 1 checking nonce 0x00000021
Thread 1 checking nonce 0x00000029
Thread 1 checking nonce 0x00000031
Thread 2 checking nonce 0x0000001a
Thread 1 checking nonce 0x00000039
Thread 2 checking nonce 0x00000022
Thread 0 checking nonce 0x00000078
Thread 0 checking nonce 0x00000080
Thread 0 checking nonce 0x00000088
Thread 0 checking nonce 0x00000090
Thread 1 checking nonce 0x00000041
Thread 2 checking nonce 0x0000002a
Thread 7 checking nonce 0x0000001f
Thread 6 checking nonce 0x00000006
Thread 2 checking nonce 0x00000032
Thread 2 checking nonce 0x0000003a
Thread 6 checking nonce 0x0000000e
Thread 6 checking nonce 0x00000016
Thread 2 checking nonce 0x00000042
Thread 2 checking nonce 0x0000004a
Thread 2 checking nonce 0x00000052
Thread 2 checking nonce 0x0000005a
Thread 2 checking nonce 0x00000062
Thread 2 checking nonce 0x0000006a
Thread 2 checking nonce 0x00000072
Thread 2 checking nonce 0x0000007a
Thread 1 checking nonce 0x00000049
Thread 7 checking nonce 0x00000027
Thread 4 checking nonce 0x00000014
Thread 1 checking nonce 0x00000051
Thread 7 checking nonce 0x0000002f
Thread 1 checking nonce 0x00000059
Thread 1 checking nonce 0x00000061
Thread 1 checking nonce 0x00000069
Thread 1 checking nonce 0x00000071
Thread 1 checking nonce 0x00000079
Thread 6 checking nonce 0x0000001e
Thread 1 checking nonce 0x00000081
Thread 7 checking nonce 0x00000037
Thread 0 checking nonce 0x00000098
Thread 1 checking nonce 0x00000089
Thread 1 checking nonce 0x00000091
Thread 1 checking nonce 0x00000099
Thread 1 checking nonce 0x000000a1
Thread 1 checking nonce 0x000000a9
Thread 1 checking nonce 0x000000b1
Thread 1 checking nonce 0x000000b9
Thread 1 checking nonce 0x000000c1
Thread 1 checking nonce 0x000000c9
Thread 1 checking nonce 0x000000d1
Thread 1 checking nonce 0x000000d9
Thread 1 checking nonce 0x000000e1
Thread 1 checking nonce 0x000000e9
Thread 1 checking nonce 0x000000f1
Thread 2 checking nonce 0x00000082
Thread 2 checking nonce 0x0000008a
Thread 6 checking nonce 0x00000026
Thread 1 checking nonce 0x000000f9
Thread 6 checking nonce 0x0000002e
Thread 6 checking nonce 0x00000036
Thread 1 checking nonce 0x00000101
Thread 6 checking nonce 0x0000003e
Thread 4 checking nonce 0x0000001c
Thread 1 checking nonce 0x00000109
Thread 4 checking nonce 0x00000024
Thread 4 checking nonce 0x0000002c
Thread 4 checking nonce 0x00000034
Thread 4 checking nonce 0x0000003c
Thread 4 checking nonce 0x00000044
Thread 4 checking nonce 0x0000004c
Thread 5 checking nonce 0x00000035
Thread 7 checking nonce 0x0000003f
Thread 7 checking nonce 0x00000047
Thread 5 checking nonce 0x0000003d
Thread 7 checking nonce 0x0000004f
Thread 5 checking nonce 0x00000045
Thread 6 checking nonce 0x00000046
Thread 7 checking nonce 0x00000057
Thread 6 checking nonce 0x0000004e
Thread 6 checking nonce 0x00000056
Thread 6 checking nonce 0x0000005e
Thread 6 checking nonce 0x00000066
Thread 6 checking nonce 0x0000006e
Thread 4 checking nonce 0x00000054
Thread 4 checking nonce 0x0000005c
Thread 4 checking nonce 0x00000064
Thread 4 checking nonce 0x0000006c
Thread 4 checking nonce 0x00000074
Thread 4 checking nonce 0x0000007c
Thread 3 checking nonce 0x0000000b
Thread 4 checking nonce 0x00000084
Thread 3 checking nonce 0x00000013
Thread 4 checking nonce 0x0000008c
Thread 2 checking nonce 0x00000092
Thread 4 checking nonce 0x00000094
Thread 4 checking nonce 0x0000009c
Thread 2 checking nonce 0x0000009a
Thread 0 checking nonce 0x000000a0
Thread 5 checking nonce 0x0000004d
Thread 5 checking nonce 0x00000055
Thread 0 checking nonce 0x000000a8
Thread 5 checking nonce 0x0000005d
Thread 0 checking nonce 0x000000b0
Thread 4 checking nonce 0x000000a4
Thread 0 checking nonce 0x000000b8
Thread 5 checking nonce 0x00000065
Thread 0 checking nonce 0x000000c0
Thread 2 checking nonce 0x000000a2
Thread 1 checking nonce 0x00000111
Thread 0 checking nonce 0x000000c8
Thread 0 checking nonce 0x000000d0
Thread 0 checking nonce 0x000000d8
Thread 0 checking nonce 0x000000e0
Thread 5 checking nonce 0x0000006d
Thread 0 checking nonce 0x000000e8
Thread 0 checking nonce 0x000000f0
Thread 5 checking nonce 0x00000075
Thread 4 checking nonce 0x000000ac
Thread 5 checking nonce 0x0000007d
Thread 1 checking nonce 0x00000119
Thread 4 checking nonce 0x000000b4
Thread 2 checking nonce 0x000000aa
Thread 1 checking nonce 0x00000121
Thread 1 checking nonce 0x00000129
Thread 4 checking nonce 0x000000bc
Thread 5 checking nonce 0x00000085
Thread 4 checking nonce 0x000000c4
Thread 5 checking nonce 0x0000008d
Thread 3 checking nonce 0x0000001b
Thread 5 checking nonce 0x00000095
Thread 0 checking nonce 0x000000f8
Thread 5 checking nonce 0x0000009d
Thread 1 checking nonce 0x00000131
Thread 2 checking nonce 0x000000b2
Thread 7 checking nonce 0x0000005f
Thread 2 checking nonce 0x000000ba
Thread 6 checking nonce 0x00000076
Thread 2 checking nonce 0x000000c2
Thread 5 checking nonce 0x000000a5
Thread 5 checking nonce 0x000000ad
Thread 5 checking nonce 0x000000b5
Thread 5 checking nonce 0x000000bd
Thread 5 checking nonce 0x000000c5
Thread 5 checking nonce 0x000000cd
Thread 5 checking nonce 0x000000d5
Thread 5 checking nonce 0x000000dd
Thread 0 checking nonce 0x00000100
Thread 1 checking nonce 0x00000139
Thread 4 checking nonce 0x000000cc
Thread 1 checking nonce 0x00000141
Thread 4 checking nonce 0x000000d4
Thread 3 checking nonce 0x00000023
Thread 2 checking nonce 0x000000ca
Thread 0 checking nonce 0x00000108
Thread 3 checking nonce 0x0000002b
Thread 2 checking nonce 0x000000d2
Thread 3 checking nonce 0x00000033
Thread 7 checking nonce 0x00000067
Thread 7 checking nonce 0x0000006f
Thread 7 checking nonce 0x00000077
Thread 7 checking nonce 0x0000007f
Thread 7 checking nonce 0x00000087
Thread 7 checking nonce 0x0000008f
Thread 2 checking nonce 0x000000da
Thread 6 checking nonce 0x0000007e
Thread 6 checking nonce 0x00000086
Thread 6 checking nonce 0x0000008e
Thread 6 checking nonce 0x00000096
Thread 7 checking nonce 0x00000097
Thread 4 checking nonce 0x000000dc
Thread 4 checking nonce 0x000000e4
Thread 4 checking nonce 0x000000ec
Thread 4 checking nonce 0x000000f4
Thread 4 checking nonce 0x000000fc
Thread 2 checking nonce 0x000000e2
Thread 2 checking nonce 0x000000ea
Thread 6 checking nonce 0x0000009e
Thread 5 checking nonce 0x000000e5
Thread 7 checking nonce 0x0000009f
Thread 7 checking nonce 0x000000a7
Thread 5 checking nonce 0x000000ed
Thread 7 checking nonce 0x000000af
Thread 7 checking nonce 0x000000b7
Thread 7 checking nonce 0x000000bf
Thread 4 checking nonce 0x00000104
Thread 1 checking nonce 0x00000149
Thread 4 checking nonce 0x0000010c
Thread 2 checking nonce 0x000000f2
Thread 4 checking nonce 0x00000114
Thread 1 checking nonce 0x00000151
Thread 7 checking nonce 0x000000c7
Thread 3 checking nonce 0x0000003b
Thread 7 checking nonce 0x000000cf
Thread 3 checking nonce 0x00000043
Thread 2 checking nonce 0x000000fa
Thread 7 checking nonce 0x000000d7
Thread 2 checking nonce 0x00000102
Thread 2 checking nonce 0x0000010a
Thread 2 checking nonce 0x00000112
Thread 2 checking nonce 0x0000011a
Thread 2 checking nonce 0x00000122
Thread 2 checking nonce 0x0000012a
Thread 2 checking nonce 0x00000132
Thread 2 checking nonce 0x0000013a
Thread 3 checking nonce 0x0000004b
Thread 5 checking nonce 0x000000f5
Thread 7 checking nonce 0x000000df
Thread 7 checking nonce 0x000000e7
Thread 5 checking nonce 0x000000fd
Thread 7 checking nonce 0x000000ef
Thread 5 checking nonce 0x00000105
Thread 7 checking nonce 0x000000f7
Thread 0 checking nonce 0x00000110
Thread 5 checking nonce 0x0000010d
Thread 0 checking nonce 0x00000118
Thread 4 checking nonce 0x0000011c
Thread 5 checking nonce 0x00000115
Thread 4 checking nonce 0x00000124
Thread 5 checking nonce 0x0000011d
Thread 4 checking nonce 0x0000012c
Thread 5 checking nonce 0x00000125
Thread 5 checking nonce 0x0000012d
Thread 4 checking nonce 0x00000134
Thread 5 checking nonce 0x00000135
Thread 4 checking nonce 0x0000013c
Thread 6 checking nonce 0x000000a6
Thread 5 checking nonce 0x0000013d
Thread 5 checking nonce 0x00000145
Thread 6 checking nonce 0x000000ae
Thread 1 checking nonce 0x00000159
Thread 0 checking nonce 0x00000120
Thread 1 checking nonce 0x00000161
Thread 0 checking nonce 0x00000128
Thread 0 checking nonce 0x00000130
Thread 1 checking nonce 0x00000169
Thread 7 checking nonce 0x000000ff
Thread 1 checking nonce 0x00000171
Thread 2 checking nonce 0x00000142
Thread 7 checking nonce 0x00000107
Thread 0 checking nonce 0x00000138
Thread 6 checking nonce 0x000000b6
Thread 3 checking nonce 0x00000053
Thread 0 checking nonce 0x00000140
Thread 6 checking nonce 0x000000be
Thread 4 checking nonce 0x00000144
Thread 0 checking nonce 0x00000148
Thread 6 checking nonce 0x000000c6
Thread 2 checking nonce 0x0000014a
Thread 5 checking nonce 0x0000014d
Thread 5 checking nonce 0x00000155
Thread 0 checking nonce 0x00000150
Thread 3 checking nonce 0x0000005b
Thread 7 checking nonce 0x0000010f
Thread 3 checking nonce 0x00000063
Thread 7 checking nonce 0x00000117
Thread 3 checking nonce 0x0000006b
Thread 3 checking nonce 0x00000073
Thread 4 checking nonce 0x0000014c
Thread 7 checking nonce 0x0000011f
Thread 4 checking nonce 0x00000154
Thread 7 checking nonce 0x00000127
Thread 7 checking nonce 0x0000012f
Thread 7 checking nonce 0x00000137
Thread 7 checking nonce 0x0000013f
Thread 7 checking nonce 0x00000147
Thread 6 checking nonce 0x000000ce
Thread 4 checking nonce 0x0000015c
Thread 6 checking nonce 0x000000d6
Thread 6 checking nonce 0x000000de
Thread 2 checking nonce 0x00000152
Thread 7 checking nonce 0x0000014f
Thread 7 checking nonce 0x00000157
Thread 0 checking nonce 0x00000158
Thread 2 checking nonce 0x0000015a
Thread 1 checking nonce 0x00000179
Thread 6 checking nonce 0x000000e6
Thread 6 checking nonce 0x000000ee
Thread 6 checking nonce 0x000000f6
Thread 6 checking nonce 0x000000fe
Thread 6 checking nonce 0x00000106
Thread 6 checking nonce 0x0000010e
Thread 6 checking nonce 0x00000116
Thread 6 checking nonce 0x0000011e
Thread 6 checking nonce 0x00000126
Thread 6 checking nonce 0x0000012e
Thread 6 checking nonce 0x00000136
Thread 6 checking nonce 0x0000013e
Thread 6 checking nonce 0x00000146
Thread 6 checking nonce 0x0000014e
Thread 6 checking nonce 0x00000156
Thread 3 checking nonce 0x0000007b
Thread 3 checking nonce 0x00000083
Thread 3 checking nonce 0x0000008b
Thread 3 checking nonce 0x00000093
Thread 3 checking nonce 0x0000009b
Thread 3 checking nonce 0x000000a3
Thread 3 checking nonce 0x000000ab
Thread 3 checking nonce 0x000000b3
Thread 3 checking nonce 0x000000bb
Thread 3 checking nonce 0x000000c3
Thread 3 checking nonce 0x000000cb
Thread 3 checking nonce 0x000000d3
Thread 3 checking nonce 0x000000db
Thread 3 checking nonce 0x000000e3
Thread 3 checking nonce 0x000000eb
Thread 3 checking nonce 0x000000f3
Thread 3 checking nonce 0x000000fb
Thread 3 checking nonce 0x00000103
Thread 3 checking nonce 0x0000010b
Thread 3 checking nonce 0x00000113
Thread 3 checking nonce 0x0000011b
Thread 3 checking nonce 0x00000123
Thread 3 checking nonce 0x0000012b
Thread 3 checking nonce 0x00000133
Thread 3 checking nonce 0x0000013b
Thread 3 checking nonce 0x00000143
Thread 3 checking nonce 0x0000014b
Thread 3 checking nonce 0x00000153
Thread 3 checking nonce 0x0000015b
0x00000155 0x009a61f7
Received event END
